nipy/labs/utils/reproducibility_measures.py

```diff
@@ -665,7 +665,7 @@ def group_reproducibility_metrics(
 
     from ..mask import intersect_masks
 
-    if ((len(variance_images) == 0) & (method is not 'crfx')):
+    if ((len(variance_images) == 0) & (method != 'crfx')):
```

Comment:
Might as well write Pythonically.

```suggestion
    if len(variance_images) == 0 and method != 'crfx':
```

Reply:
You’re right, I was laser-focused on the particular fix and didn’t review the surrounding line.

I guess the even more Pythonic form would be:

```python
if not variance_images and method != 'crfx':
```

Reply:
> ```python
> if not variance_images and method != 'crfx':
> ```

Which form should I change it to?

Reply:
`len()` makes it clearer to a reader that a list or tuple or similar is expected, so I'd go with that.